sklearn/linear_model/tests/test_bayes.py

```diff
@@ -209,7 +210,8 @@ def test_ard_accuracy_on_easy_problem(global_random_seed, n_samples, n_features)
     assert abs_coef_error < 1e-10
 
 
-def test_return_std():
+@pytest.mark.parametrize("as_frame", [True, False])
```

Comment:
Actually, we could use the tool `sklearn.utils._testing.convert_container` that embed already the `importorskip`

```python
@pytest.mark.parametrize("constructor_name", ["array", "dataframe"])
```